pylib/cqlshlib/cql3handling.py

```diff
@@ -1390,21 +1390,21 @@ def idx_ks_idx_name_completer(ctxt, cass):
 
 
 syntax_rules += r'''
-<alterTableStatement> ::= "ALTER" wat=( "COLUMNFAMILY" | "TABLE" ) cf=<columnFamilyName>
+<alterTableStatement> ::= "ALTER" wat=( "COLUMNFAMILY" | "TABLE" ) ("IF" "EXISTS")? cf=<columnFamilyName>
```

Comment:
Mmmm we don't seem to have an later MV stmnt, weird... Food for another ticket I guess.

Reply:
Yes, 😄 an opportunity for another ticket to add _alter MV stmt in cqlsh_. 👍

Reply:
You want me to create it or do you want to have a go yourself?

Reply:
Created it here https://issues.apache.org/jira/browse/CASSANDRA-17447.